rest/provider_test.go

```diff
@@ -212,18 +212,18 @@ func TestDiffForUpdateableResource(t *testing.T) {
 	ctx := context.Background()
 
 	oldInputsJSON := `{
```

Comment:
The naming scheme data used for this test was incorrect - the `oldInputsJSON` and `newInputsJSON` keys need to use the SDK naming scheme (not in the API naming scheme) - in this test they are standing in as resource state inputs, not data from the provider API.